src/controllers/expenses.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+const userService = require('../services/users');
+const expensesService = require('../services/expenses');
+
+const getAllExpenses = (req, res) => {
+  const {
+    userId,
+    categories,
+    from,
+    to,
+  } = req.query;
+
+  let expenses = expensesService.getAllExpenses();
+
+  if (userId) {
+    expenses = expenses.filter(expense => expense.userId === +userId);
+  }
+
+  if (categories) {
+    expenses = expenses.filter(({ category }) => (
+      categories.includes(category)
+    ));
+  }
+
+  if (from && to) {
+    expenses = expenses.filter(expense => {
+      const expenseDate = new Date(expense.spentAt);
+      const fromDate = new Date(from);
+      const toDate = new Date(to);
+
+      return expenseDate < toDate && fromDate <= expenseDate;
+    });
+  };
+
+  res.send(expenses);
+};
+
+const getOneExpence = (req, res) => {
+  const { expenseId } = req.params;
+  const foundExpens = expensesService.findExpensesById(+expenseId);
```

Comment:
Pay attention to the naming
Also, would be better to use `Number()` instead `+`
```suggestion
  const foundExpense = expensesService.findExpensesById(Number(expenseId));
```